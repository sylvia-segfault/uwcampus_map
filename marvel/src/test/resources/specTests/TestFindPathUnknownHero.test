# test the edge case when the given hero is not
# in this graph

LoadGraph graph biggerData.tsv
FindPath graph happy coding