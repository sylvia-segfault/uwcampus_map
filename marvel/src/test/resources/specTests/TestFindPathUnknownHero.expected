# test the edge case when the given hero is not
# in this graph

loaded graph graph
unknown character happy
unknown character coding